tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -244,6 +244,45 @@ static struct device_info boards[] = {
 		.first_sysupgrade_partition = "os-image",
 		.last_sysupgrade_partition = "support-list",
 	},
+	/** Firmware layout for the CPE610 */
+	{
+		.id     = "CPE610",
```

Comment:
Any chance of having this changed to CPE610V1? If yes, same would be necessary in generic-tp-link.mk.

Reply:
I will upload an update with your suggestions after testing it.